types/conv.go

```diff
@@ -57,8 +57,8 @@ func ToABCIHeader(header *Header) tmtypes.Header {
 		},
 		LastCommitHash:     header.LastCommitHash[:],
 		DataHash:           header.DataHash[:],
-		ValidatorsHash:     header.SequencersHash[:],
-		NextValidatorsHash: header.SequencersHash[:],
+		ValidatorsHash:     header.NextSequencersHash[:],
+		NextValidatorsHash: header.NextSequencersHash[:],
 		ConsensusHash:      header.ConsensusHash[:],
```

Comment:
I don't understnad why both the same

Reply:
those values actually not used by dymint for anything.
it was empty set till now.

Reply:
not sure how IBC treats it thou. and how it worked till now

Reply:
anyway i'll change it, so the `ValidatorsHash` will be correct